src/App.jsx

```diff
@@ -14,70 +15,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [activeGood, setActiveGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const Good = ({ good }) => {
```

Comment:
don't define one component inside another
create a separate file for `Good` component and import it here
Pass `activeGood` as prop to the `Good` component

Reply:
ok, thanks, I understood, but there was one requirement - do all inside App without external components, that's why I will just delete this component